CMakeLists.txt

```diff
@@ -164,6 +164,53 @@ function (monero_add_minimal_executable name)
     monero_set_target_no_relink( ${name} )
 endfunction()
 
+function(forbid_undefined_symbols)
+  add_linker_flag_if_supported(-Wl,--no-undefined TMP)
+  add_linker_flag_if_supported(-Wl,-udefined,error TMP)
```

Comment:
maybe a typo ? Not sure what linker this is for.

Reply:
Thanks, updated. It wasn't caught because `-undefined error` is default on Mac anyway.